### PR TITLE
Upgrade to Lucene 9.7.0.

### PR DESCRIPTION
**Pull request by jpountz**
This is the official release, and is very close to the snapshot we're currently on.

### CODE REVIEW

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -2430,7 +2458,9 @@
       </component>
       <component group="org.apache.kerby" name="ldap-backend" version="1.1.1">
          <artifact name="ldap-backend-1.1.1.jar">
-            <sha256 value="d3c030fe9eaeeb8a7b0844a1b48cecef20bc0a8deb5718a6c482bfc325069ee0" origin="Generated by Gradle"/>
+            <sha256 value="d3c030fe9eaeeb8a7b0844a1b48cecef20bc0a8deb5718a6c482bfc325069ee0" origin="Generated by Gradle">
+               <also-trust value="b427406dbe76279cf4fd496af17792c4c39fb6f295480290bb8be987d33cbc20"/>
```
Comment (kingherc) [inline gradle/verification-metadata.xml:2462]: Why do we need also-trust values?

**Review by kingherc — COMMENTED** [review]
CI seems to be failing

File: gradle/verification-metadata.xml (hunk 1 above)
Comment (jpountz) [reply gradle/verification-metadata.xml:2462]: `gradlew updateShas` did this for me, I don't know why. @breskeby Do you know about this?

**Comment by jpountz** [comment]
> CI seems to be failing

Indeed, not all artifacts were available on Maven Central yet based on the output, I'll rerun tests.

@elasticsearchmachine test this please

**Comment by jpountz** [comment]
It looks like it's more complicated than that, I'll dig.

**Comment by jpountz** [comment]
Re-computing verification metadata did the trick, it's green now.

**Review by ChrisHegarty — APPROVED** [review]
LGTM.

File: gradle/verification-metadata.xml [hunk 2]
```diff
@@ -2488,121 +2488,241 @@
             <sha256 value="015d5c229f3cd5c0ebf175c1da08d596d94043362ae9d92637d88848c90537c8" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.lucene" name="lucene-analysis-common" version="9.7.0">
+         <artifact name="lucene-analysis-common-9.7.0.jar">
+            <sha256 value="87abf0221c526d64616d47d2261866620c8f42659c93dbfd0d1ffb138af98137" origin="Generated by Gradle"/>
+         </artifact>
```
Comment (iverase) [inline gradle/verification-metadata.xml:2494]: We should remove the old entries, unfortunately the command don't do it for us and we should prevent the file to become huge

Comment (iverase) [reply gradle/verification-metadata.xml:2494]: than you so much Adrien!

**Review by iverase — APPROVED** [review]
LGTM

**Comment by iverase** [comment]
@jpountz in case you miss it, it seems backport bot didn't work this time.

**Comment by jpountz** [comment]
I did miss it, thanks @iverase!

**Pull request merged**